runtime/compiler/ilgen/J9ByteCodeIlGenerator.hpp

```diff
@@ -379,6 +389,11 @@ class TR_J9ByteCodeIlGenerator : public TR_IlGenerator, public TR_J9ByteCodeIter
    // JSR292
    int32_t                           _argPlaceholderSlot;
    int32_t                           _argPlaceholderSignatureOffset;
+   TR::NodeChecklist                *_methodHandleInvokeNodes;
+   TR::NodeChecklist                *_invokeHandleNodes;
+   TR::NodeChecklist                *_invokeHandleGenericNodes;
+   TR::NodeChecklist                *_invokeDynamicNodes;
+   TR::NodeChecklist                *_ilGenMacroInvokeExactNodes;
```

Comment:
Using `NodeChecklist` for this will break if a node of interest is duplicated, e.g. because of a catch block catching multiple exception types. This happened before with `expandInvokeSpecialInterface()`, which is why `_invokeSpecialInterfaceCalls` identifies the calls by their bytecode indices.